src/Parsers/ASTColumnsTransformers.h

```diff
@@ -25,13 +25,22 @@ class ASTColumnsApplyTransformer : public IASTColumnsTransformer
         auto res = std::make_shared<ASTColumnsApplyTransformer>(*this);
         if (parameters)
             res->parameters = parameters->clone();
+        if (lambda)
+            res->lambda = lambda->clone();
```

Comment:
Should we also add `res->lambda_arg = lambda_arg;` ?

Reply:
Nope. String is deep cloned by default.